interop/quic-openssl-docker/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM martenseemann/quic-network-simulator-endpoint:latest
+
+# Make sure curl picks up the new openssl
+ENV PKG_CONFIG_LIBDIR=/usr/lib64/pkgconfig:/usr/lib/pkgconfig:/usr/lib/x86_64-linux-gnu/pkgconfig/:$PKG_CONFIG_LIBDIR
+# Set the environment variable LD_LIBRARY_PATH to ensure we get the right libraries
+ENV LD_LIBRARY_PATH=/usr/lib64:/usr/lib:/usr/lib/x86_64-linux-gnu:$LD_LIBRARY_PATH
+
+# Install needed tools
+RUN apt-get update && apt-get install -y \
+    git make gcc perl cmake build-essential \
+    autoconf libtool pkg-config libpsl-dev
+
+# build nghttp3
+RUN git clone https://github.com/ngtcp2/nghttp3.git && \
+    cd nghttp3 && \
+    git submodule update --init && \
+    autoreconf -i && \
+    ./configure --prefix=/usr && \
+    make -j check && \
```

Comment:
I think `-j` should be followed by number of jobs.

Reply:
in gnu make -j with no argument indicates no limit to parallelism.  No issue with limiting that though.  It will just take a bit longer to build.

Reply:
OK I did not know that. so leave it as is.

Reply:
set jobs to 4 in all builds